modules/builds-running-entitled-builds-with-sharedsecret-objects.adoc

```diff
@@ -61,15 +62,47 @@ rules:
       - use
 EOF
 ----
-
++
 . Create the `RoleBinding` associated with the role by using the `oc` command:
 +
 .Example `oc create rolebinding` command
 [source,terminal]
 ----
 $ oc create rolebinding shared-resource-my-share --role=shared-resource-my-share --serviceaccount=my-namespace:builder
 ----
++
+. Add a volume to mount the `rhsm.conf` file into the container. Specify the path to `rhsm.conf` as the source for this volume.
++
+[TIP]
+=== 
+You must have access to Red Hat entitlements through your subscription. The entitlement secret is automatically created by the Insights Operator. When you perform an Entitlement Build using Red Hat Enterprise Linux (RHEL) 7, you must have the following instructions in your Dockerfile before you run any yum commands:
```

Comment:
Please check the terminology - my understanding is that the cluster is _subscribed_. Once the cluster is subscribed, the entitlements are granted and synced via the Insights Operator.

```suggestion
Your cluster must be attached to an active Red Hat subscription. The entitlement secret is automatically created by the Insights Operator.

When using RHEL 7 or UBI8 as the base image of your build, you must add the following instructions to your Dockerfile before executing any `yum` or `dnf` commands:
```